packages/desktop-client/src/components/modals/EditRule.js

```diff
@@ -135,7 +134,7 @@ function EditorButtons({ onAdd, onDelete, style }) {
           style={{ padding: 7 }}
           aria-label="Add entry"
         >
-          <AddIcon style={{ width: 10, height: 10 }} />
+          <AddIcon style={{ width: 10, height: 10, color: 'inherit' }} />
```

Comment:
I pushed a commit that changes ALL icons to inherit their color, since this is what we need to do 99% of the time. Eventually we can remove all `color: 'inherit'` on icons.